processor/spanmetricsprocessor/config.go

```diff
@@ -45,4 +45,8 @@ type Config struct {
 	// The dimensions will be fetched from the span's attributes. Examples of some conventionally used attributes:
 	// https://github.com/open-telemetry/opentelemetry-collector/blob/main/model/semconv/opentelemetry.go.
 	Dimensions []Dimension `mapstructure:"dimensions"`
+
+	// MetricKeyToDimensionsCacheSize defines the size of the LRU cache for storing MetricKeyToDimensions cache, which
+	// helps to avoid cache memory growing indefinitely over the lifetime of the collector.
+	MetricKeyToDimensionsCacheSize int `mapstructure:"metric_key_to_dimensions_cache_size"`
```

Comment:
This is really verbose for a configuration setting and I am not sure I would grok that on the first read through.
Could you simplify the name here to make it easier for new users to understand what and why they would want to set this variable?

Reply:
yeah, I agree. We don't really need to expose the implementation detail as this is a cache of `MetricKeyToDimensions`.